### DISCUSSION TITLE
No matching distribution found for mxnet-cu102 on windows systems

### DISCUSSION THREAD

**Discussion by codealphago** [apache/incubator-mxnet#19311] [Q&A] [answered]
Hi!I have a problem installing mxnet on windows systems.The console output:"No matching distribution found for mxnet-cu102"

**Comment by szha** [comment]
cc @yajiedesign. The problem has been that the wheels for Windows are too large to upload to pypi.

**Comment by dai-ichiro** [comment, marked as answer]
You can install nightly release.
`pip install mxnet-cu102==1.7.0 -f https://dist.mxnet.io/python/cu102`

**Reply by codealphago** [reply]
Thanks!it works.
